crashsimilarity/crash_similarity.py

```diff
@@ -10,14 +10,19 @@
 import gensim
 import numpy as np
 import pyximport
-from pyemd import emd
 
 from crashsimilarity import utils
 from crashsimilarity.downloader import SocorroDownloader
 from crashsimilarity.utils import StackTraceProcessor
 
 pyximport.install()
 
+try:
+    from pyemd import emd
```

Comment:
Let's make this package strictly required as it was before, it's in the requirements.txt file anyway